provider/provider_test.go

```diff
@@ -402,7 +409,8 @@ func configureProviderAndReturnClient(t *testing.T, tt providerFixture) (*common
 		return nil, fmt.Errorf(strings.Join(issues, ", "))
 	}
 	client := p.Meta().(*common.DatabricksClient)
-	err := client.Authenticate(ctx)
+	req, _ := http.NewRequest("GET", client.Config.Host, nil)
+	err := client.Config.Authenticate(req)
```

Comment:
I think this should happen in `configureDatabricksClient` to ensure parity with existing behavior.